applications/HelloIOIO/build.gradle

```diff
@@ -39,7 +39,7 @@ dependencies {
     codeImplementation project(':IOIOLibAndroidAccessory')
     codeImplementation project(':IOIOLibAndroidDevice')
 
-    def ioioVersion = "6.0.0-beta5"
+    def ioioVersion = "6.0.0.beta7"
```

Comment:
As mentioned in another PR, can we pull this in programmatically to test the most recent jitpack build? I.e. using something like your [gittag()](https://github.com/ytai/ioio/blob/dbf6b4187b0c3d32cca67d583ce52e343fced61e/build.gradle#L36) method? We'd have to modify it to pull the most recent -1 tag somehow else it would be a circular definition, but I guess this can be done. 

I can try to implement it, but want to know if you see any obvious reason this would not work well.

Reply:
I see nothing against it

Reply:
The thing can be very complicated, when you change something, which is in last version different.
I would go by simply remove flavor

Reply:
I agree that removing the flavors simplifies things a lot, and I'm trying to prove to myself that we are not losing any checking functionality by removing this. When we build the library build flavor of HelloIOIO on a previous jitpack build (E.g 6.0.0.Beta8), we are getting no checks that we would not already get from building the previous jitpack based on the source code used to compile 6.0.0.Beta8 right?

Reply:
Yes, here we test only on an old version